Source/ge_pz/MainGameMode.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
+
+#include "MainGameMode.h"
+#include "ge_pzCharacter.h"
+#include "MainPlayerController.h"
+#include "UObject/ConstructorHelpers.h"
+
+AMainGameMode::AMainGameMode()
+{
+	// set default pawn class to our Blueprinted character
+	static ConstructorHelpers::FClassFinder<APawn> PlayerPawnBPClass(TEXT("/Game/ThirdPersonCPP/Blueprints/BP_ThirdPersonCharacter"));
+	if (PlayerPawnBPClass.Class != NULL)
```

Comment:
Рекомендую использовать coding style от эпиков. 
> nullptr should be used instead of the C-style NULL macro in all cases.

https://docs.unrealengine.com/4.27/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/